pkg/index/validate.go

```diff
@@ -93,6 +96,9 @@ func (p Platform) Validate() error {
 	if p.Sha256 == "" {
 		return errors.New("sha256 sum has to be set")
 	}
+	if !isValidSHA256(p.Sha256) {
+		return errors.Errorf("sha256 value %s is not a valid sha256sum string", p.Sha256)
```

Comment:
Could you mention the requirement, that the string is expected to be lower-case?

Reply:
good point.

Reply:
I decided to include the pattern in the error message.